### PR TITLE
fix(ui): Ensure QuickTrace takes full width in the UI

### PR DESCRIPTION
**Pull request by evanpurkhiser**
here's what it looked like before

image

and now

image

of course, the padding is still wrong. But I'll clean that up separately.

### CODE REVIEW

File: static/app/views/organizationGroupDetails/eventToolbar.tsx [hunk 1]
```diff
@@ -120,87 +120,92 @@ class GroupEventToolbar extends Component<Props> {
...
-            </LinkContainer>
-          </Heading>
-          <Tooltip title={this.getDateTooltip()} showUnderline disableForVisualTest>
-            <StyledDateTime
-              format={is24Hours ? 'MMM D, YYYY HH:mm:ss zz' : 'll LTS z'}
-              date={getDynamicText({
-                value: evt.dateCreated,
-                fixed: 'Dummy timestamp',
-              })}
+        <EventWrapper>
```
Comment (priscilawebdev) [inline static/app/views/organizationGroupDetails/eventToolbar.tsx:123]: was not possible to only update the styles from the `Wrapper` component?

File: static/app/views/organizationGroupDetails/eventToolbar.tsx [hunk 2]
```diff
@@ -120,87 +120,92 @@ class GroupEventToolbar extends Component<Props> {
...
+          <div>
```
Comment (priscilawebdev) [inline static/app/views/organizationGroupDetails/eventToolbar.tsx:124]: do we need this `div`?

**Review by priscilawebdev — APPROVED** [review]
left some questions but it looks good to me..thanks for the screenshots

**Comment by evanpurkhiser** [comment]
This breaks something else, damn

**Comment by evanpurkhiser** [comment]
image

**Comment by priscilawebdev** [comment]
we should have a visual check for this 🤔

**Comment by evanpurkhiser** [comment]
Good call. I'll see what I can do

**Pull request closed without merging**